bindata/v3.11.0/kube-controller-manager/pod.yaml

```diff
@@ -21,9 +21,25 @@ spec:
       requests:
         memory: 200Mi
         cpu: 100m
+    ports:
+      - containerPort: 10257
     volumeMounts:
     - mountPath: /etc/kubernetes/static-pod-resources
       name: resource-dir
+    livenessProbe:
+      httpGet:
+        scheme: HTTPS
+        port: 10257
+        path: healthz
+      initialDelaySeconds: 45
```

Comment:
copied from kube-apiserver, i'm up for suggestion on better numbers (i think we need some initial delay till the controller manager apiserver boots up)